MetadataExtractor/Formats/Exif/ExifDescriptorBase.cs

```diff
@@ -251,19 +251,6 @@ public string GetYCbCrPositioningDescription()
         }
 
         [CanBeNull]
-        public string GetOrientationDescription()
```

Comment:
It's worth keeping this method for consistency with other `Get*Description` methods, though it could delegate to the version on `TagDescriptor`.